lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart

```diff
@@ -5,16 +5,49 @@ import 'package:get/get.dart';
 import 'package:tmail_ui_user/features/email/presentation/email_view.dart';
 import 'package:tmail_ui_user/features/mailbox/presentation/mailbox_view.dart';
 import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/base_mailbox_dashboard_view.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart';
 import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/model/dashboard_routes.dart';
 import 'package:tmail_ui_user/features/search/presentation/search_email_view.dart';
 import 'package:tmail_ui_user/features/thread/presentation/thread_view.dart';
 
 class MailboxDashBoardView extends BaseMailboxDashBoardView {
-
   MailboxDashBoardView({Key? key}) : super(key: key);
 
+  final mailBoxDashBoardController = Get.find<MailboxDashBoardController>();
+
   @override
   Widget build(BuildContext context) {
+    return Obx(() {
+      return Scaffold(
+        key: controller.scaffoldKey,
+        drawer: ResponsiveWidget(
+            responsiveUtils: responsiveUtils,
+            mobile: SizedBox(child: MailboxView(), width: double.infinity),
+            landscapeMobile: SizedBox(
+                child: MailboxView(), width: ResponsiveUtils.defaultSizeDrawer),
+            tablet: SizedBox(
+                child: MailboxView(), width: ResponsiveUtils.defaultSizeDrawer),
+            landscapeTablet: SizedBox(
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile),
+            tabletLarge: SizedBox(
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile),
+            desktop: SizedBox(
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile)),
+        drawerEnableOpenDragGesture:
+            responsiveUtils.hasLeftMenuDrawerActive(context) &&
+                !(mailBoxDashBoardController.dashboardRoute.value ==
```

Comment:
1. Please double check the email search case.
2. Test it on a web browser when resizing the screen.

Reply:
please provide demo @sherlockvn